### DISCUSSION TITLE
奇怪的塔状logo

### DISCUSSION THREAD

**Discussion by SSTV-10068** [share121/inter-knot#1049] [其他]
近日某些新艾利都居民发现了一些身着印着塔状logo怪异服装的怪异人士。其实这是我司正在开发的一款游戏《明日之舟》的周边服饰，是公司内部的福利套装，所以请各位不要担心，只是某些员工穿着出去逛街罢了。
所以请各位放心，在此，我司向各位感到抱歉。
所以我司一定会管理好手下的员工，防止此种事件再次发生。
![356811022-1bf32f01-9e87-4060-aa4e-ef4b0e5588c0](https://github.com/user-attachments/assets/099e2cf7-d779-496e-aa73-12705bc63fa9)

**Comment by llxcx4** [comment]
什么，粥！何时来的₍•Д•)

**Comment by Kamisato100** [comment]
什么，舟吧打过来了！

**Comment by Pasukalu** [comment]
₍•Д•)

**Comment by xioagong** [comment]
完了，文化入侵

**Comment by xianyujun4** [comment]
什么？！是方舟？！快！城防炮填充活性源石，放！！！

**Comment by lllliufeng** [comment]
Σ(⊙▽⊙"a

**Comment by hezixianyu** [comment]
罗德岛生物制药公司，小到区域战争，大到种族灭绝，欢迎您的委托。

**Comment by ghhccghk** [comment]
罗德岛现在还招人吗
